build.gradle.kts

```diff
@@ -328,11 +328,7 @@ tasks {
       commandLine("pnpm", "run", "build:agent")
     }
     copy {
-      from(agentDir.resolve("dist")) {
```

Comment:
Currently we copy all files form that dir, there is no reason to pick them one by one.